cubiscan/command.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2019 Camptocamp SA
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html)
+"""Defines a Registry for Commands and their Responses."""
+
+from cubiscan import base_responses
+
+COMMAND_REGISTRY = None
+
+
+class CommandRegistry(object):
+
+    command_bits = None
+    response_mappings = None
+
+    def __init__(self):
+        self.command_bits = {}
+        self.response_mappings = {}
+        self.init_base_mappings()
+
+    def init_base_mappings(self):
+        """Register the base commands."""
+        self.add_command(
+            'continous_measure', bytes('C', 'ascii'),
+            base_responses.MEASURE, base_responses.NEG_MEASURE
+        )
+        self.add_command(
+            'dim_calibration', bytes('D', 'ascii'),
+            base_responses.DIM_CALIBRATION, []
+        )
+        self.add_command('dim_unit', bytes('"', 'ascii'), [], [])
+        self.add_command('set_factor', bytes('F', 'ascii'), [], [])
+        self.add_command('location', bytes('L', 'ascii'), [], [])
+        self.add_command(
+            'measure', bytes('M', 'ascii'),
+            base_responses.MEASURE, base_responses.NEG_MEASURE
+        )
+        self.add_command(
+            'weight_calibration', bytes('S', 'ascii'),
+            base_responses.WEIGHT_CALIBRATION, []
+        )
+        self.add_command(
+            'test', bytes('T', 'ascii'), base_responses.TEST, []
+        )
+        self.add_command(
+            'units', bytes('U', 'ascii'), base_responses.REPORT_UNITS, []
+        )
+        self.add_command('weight_unit', bytes('#', 'ascii'), [], [])
+        self.add_command('zero', bytes('Z', 'ascii'), [], [])
+
+    def add_command(self, name, command_bit, response, neg_response):
```

Comment:
I figured that another version might have more commands. Also i excluded the ones to change ip, subnetmask and gateway since it can stop you from having access to the device. but someone might want it anyway but otherwise yes i could just have a mapping.